opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -41,6 +65,25 @@ impl UserEventsExporter {
             );
         }
 
+        Ok(UserEventsOptions { provider_name })
+    }
+}
+
+/// UserEventsExporter is a log exporter that exports logs in EventHeader format to user_events tracepoint.
+pub struct UserEventsExporter {
```

Comment:
not required to be public, right?

Reply:
Needed to be public for invoking `UserEventsExporter ::build_processor()`

Reply:
`build_processor()` method doesn't require `&self` so we should be able to move it outside of `impl UserEventsExporter` block and mark `UseEventsExporter` struct internal.

Reply:
@cijothomas Unresolved this comment as `UserEventsExporter` struct need not be public.

Reply:
Done, though I would have preferred to have `build_processor` scoped as static method under `UserEventsExporter`. Not, we didn't expose any of the member variables under UserEventsExporter as public.